src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -0,0 +1,332 @@
+/** GeoMap Visualization. */
+
+// =================
+// === Constants ===
+// =================
+
+/**
+ * Mapbox API access token.
+ * All the limits of API are listed here: https://docs.mapbox.com/api/#rate-limits
+ */
+const TOKEN =
+  "pk.eyJ1IjoiZW5zby1vcmciLCJhIjoiY2tmNnh5MXh2MGlyOTJ5cWdubnFxbXo4ZSJ9.3KdAcCiiXJcSM18nwk09-Q";
+const GEO_POINT = "Geo_Point";
+const GEO_MAP = "Geo_Map";
+const SCATTERPLOT_LAYER = "Scatterplot_Layer";
+const DEFAULT_POINT_RADIUS = 150;
+
+const DEFAULT_MAP_ZOOM = 11;
+const DARK_ACCENT_COLOR = [222, 162, 47];
+const LIGHT_ACCENT_COLOR = [1, 234, 146];
+
+// =====================================
+// === Script & Style Initialisation ===
+// =====================================
+
+function loadScript(url) {
+  var script = document.createElement("script");
+  script.src = url;
+
+  document.head.appendChild(script);
+}
+
+function loadStyle(url) {
+  var link = document.createElement("link");
+  link.href = url;
+  link.rel = "stylesheet";
+
+  document.head.appendChild(link);
+}
+
+loadScript("https://unpkg.com/deck.gl@latest/dist.min.js");
+loadScript("https://api.tiles.mapbox.com/mapbox-gl-js/v1.6.1/mapbox-gl.js");
+loadStyle("https://api.tiles.mapbox.com/mapbox-gl-js/v1.6.1/mapbox-gl.css");
+
+const styleHead = document.createElement("style");
+styleHead.innerText = `.mapboxgl-map {
+            border-radius: 14px;
+        }`;
```

Comment:
this is a very strange alignment. Either make it one line, or move the first line to next line and do proper indentation